README.md

```diff
@@ -1,45 +1,40 @@
-  ```
-  IBM Cloud Dashboard -> <your cluster> -> Access Tab
-  ```
-* Create (if not already) namespace on IBM Cloud container registry. It can be accessed at:
-  ```
-  IBM Cloud Dashboard -> Click on Navigation Menu -> Kubernetes -> Registry -> Namespaces
-  ```
-* Configure [Git CLI](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Clone the repository using the command below:
+* Create an [IBM Cloud](https://cloud.ibm.com/login) account.
+* Get an instance of [Kubernetes Service on IBM Cloud](https://cloud.ibm.com/kubernetes/catalog/cluster), which should take approximately 20 minutes.
+* Access a Kubernetes Cluster through the `kubectl` CLI. To access the instructions, go to **IBM Cloud dashboard > [your cluster] > Access**.
+* Create a namespace on IBM Cloud container registry. To do so, go to  your IBM Cloud dashboard and click **Navigation > Kubernetes > Registry > Namespaces**.
+* Configure the [Git CLI](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Clone the repository to your workstation using the following command:
+ 
   ```
   git clone https://github.com/IBM/deploy-app-using-tekton-on-kubernetes.git
   ```
   
-  > Note: You should clone this repository to your workstation since you need to edit some of the files before using them.
-  
-## Estimated Time
-This tutorial takes about 40 minutes, after pre-requisites configuration.
+## Estimated time
 
-## Section 1 - To build and deploy an application on IBM Cloud Kubernetes Service using kubectl
+After your prerequisites are configured, this tutorial takes about 40 minutes.
 
-The following steps are performed to build and deploy an application on Kubernetes cluster.
+## Build and deploy an application on IBM Cloud Kubernetes Service using `kubectl`
 
--	Write a Dockerfile for your application and build the container image using Dockerfile.
--	Upload the built container image to the accessible container registry.
--	Create a Kubernetes deployment using the container image and deploy the application to an IBM Cloud Kubernetes Service cluster using configuration(yaml) files.
+You perform the following steps to build and deploy an application on a Kubernetes cluster.
 
-For this tutorial, we have taken the simple hello-world nodejs application to deploy on Kubernetes as shown.
+1.	Write a Dockerfile for your application and build the container image using Dockerfile.
+1. 	Upload the built container image to the accessible container registry.
+1.	Create a Kubernetes deployment using the container image and deploy the application to an IBM Cloud Kubernetes Service cluster using configuration (yaml) files.
+
+For this tutorial, we have taken a simple "Hello world!" Node.js application to deploy on Kubernetes as shown.
+<!--EM: Where does this Node.js Application come from? Is that cloned from somewhere? Where does Appsody come into play here? We haven't mentioned it before. I guess I dont' really undertsand where this code is coming from.-->
```

Comment:
The sample nodejs code - created using Appsody. Appsody is one of the tool in Kabanero framework. I have talked about this in the end of the tutorial. 
As I do not want to introduce appsody here and it is just few lines of code so I showed code as a code block. Users can use any sample nodejs code if they are willing to.

Reply:
Okay, so in the tutorial, to follow along, do the readers need to have a sample app they're working with? Sorry I'm not following . . .

Reply:
Not actually. But they can use if user is not using the sample provided in this repository.